frontendDataKeys/src/main/java/com/virtuslab/gitmachete/frontend/datakeys/DataKeys.java

```diff
@@ -18,14 +19,13 @@ private DataKeys() {}
 
   public static final DataKey<IBranchLayout> KEY_BRANCH_LAYOUT = DataKey.create("BRANCH_LAYOUT");
   public static final DataKey<Path> KEY_GIT_MACHETE_FILE_PATH = DataKey.create("GIT_MACHETE_FILE_PATH");
-  public static final DataKey<IGraphTableManager> KEY_GRAPH_TABLE_MANAGER = DataKey.create("GRAPH_TABLE_MANAGER");
   public static final DataKey<IGitMacheteRepository> KEY_GIT_MACHETE_REPOSITORY = DataKey.create("GIT_MACHETE_REPOSITORY");
-  public static final DataKey<Boolean> KEY_IS_GIT_MACHETE_REPOSITORY_READY = DataKey.create("IS_GIT_MACHETE_REPOSITORY_READY");
+  /** This key must always be available in the container hierarchy, and a DataProvider must always return a non-null value. */
+  public static final DataKey<@NonNull IGraphTableManager> KEY_GRAPH_TABLE_MANAGER = DataKey.create("GRAPH_TABLE_MANAGER");
```

Comment:
Is `@NonNull` required here?

Reply:
Nullness check behaves really weird with generics, esp. the ones defined outside of Checker-processed codebase... I'm just adding annotations defensively ;D without that `@NonNull` (and. esp without `@Nullable` for all other `DataKeys` here, which I've also added), Checker isn't going to raise certain errors and/or will raise them where unneeded :/

Reply:
I tried to circumvent that somehow with another custom subtyping hierarchy, but this time to no avail :/